config/packages/prod/doctrine.yml

```diff
@@ -1,12 +1,8 @@
 doctrine:
     dbal:
-        driver:   pdo_pgsql
-        host:     '%env(DATABASE_HOST)%'
-        port:     '%env(DATABASE_PORT)%'
-        dbname:   '%env(DATABASE_NAME)%'
-        user:     '%env(DATABASE_USER)%'
-        password: '%env(DATABASE_PASSWORD)%'
+        driver: pdo_pgsql
+        url: '%env(resolve:DATABASE_URL)%'
+        server_version: '%env(resolve:DATABASE_SERVER_VERSION)%'
```

Comment:
in new sf4 projects i pass server version as part of database url: `?charset=utf8&serverVersion=10`